ethereum/contracts/Implementation.sol

```diff
@@ -32,6 +32,31 @@ contract Implementation is Governance {
 
     function initialize() initializer public virtual {
         // this function needs to be exposed for an upgrade to pass
+        uint256 evmChainId;
+        uint16 chain = chainId();
+
+        // Wormhole chain ids explicitly enumerated
```

Comment:
we should wrap this whole thing in a
```
if (evmChainId == 0) {...}
```
so that we can keep this code around for longer and run it for newer deployments if needed (if those need to be updated too for some other reason).

Reply:
changed